clients/web/src/views/layout/FooterView.js

```diff
@@ -9,9 +9,16 @@ import 'girder/stylesheets/layout/footer.styl';
  * This view shows the footer in the layout.
  */
 var LayoutFooterView = View.extend({
+
+    initialize: function (settings) {
+        var mail = settings.contactEmail; // Has to change
```

Comment:
I'm not sure, but maybe not. If a default is needed for this widget, it means that it's being used in another application that didn't pass enough parameters at all.

Maybe if no user is passed at all, then don't use an `href`? What do you think?